stan/math/prim/scal/meta/scalar_seq_view.hpp

```diff
@@ -33,7 +38,7 @@ class scalar_seq_view<
   int size() const { return c_.size(); }
 
  private:
-  C c_;
+  math::plain_type_keep_constness_and_ref_t<C> c_;
```

Comment:
I think we just want to keep the const and not the ref and that should fix the current thing on jenkins, guessing tho just from the stackoverflow post below 

https://stackoverflow.com/questions/38226443/const-reference-to-member-of-temporary-object

Reply:
Also we should keep meta type traits stuff in the stan namespace and not math, that's where the rest of the stuff is

Reply:
We are actually [not terribly consistent](https://github.com/stan-dev/math/issues/1122#issuecomment-468953210) with the namespaces, tho the majority is in stan::.

Reply:
> and not the ref 
We definitely want ref to avoid unnecessary copys whan a lvalue is passed to constructor. I am not extending lifetime of an object with a const ref (which is what that SO question is about). I just want to keep ref to the object that is passed in without making any deep copys. And I am running out of ideas how to achieve that. 

Afaik `scalar_seq_view` is only constructed from lvalues. I only used perfect forwarding in attempt to work around the warning this triggers in clang. Btw this is valid code gcc is happy with. However with -Wall that warning fails compilation with clang.

About that namespace. I agree with consistancy point. However such details can wait until code works.

Reply:
Word!

For the namespace thing we are def super dodgy on it. When I see meta stuff in math and not stan I try to fix it up but we def lack consistency